### PR TITLE
Challenge: AbdallahRdf submission

### PR DESCRIPTION
**Pull request by AbdallahRdf**

### CODE REVIEW

File: submissions/AbdallahRdf/Main.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import java.io.BufferedReader;
```
Comment (aboullaite) [inline submissions/AbdallahRdf/Main.java:1]: Can you merge them into one file

**Review by aboullaite — CHANGES_REQUESTED** [review]

File: submissions/AbdallahRdf/Main.java (hunk 1 above)
Comment (AbdallahRdf) [reply submissions/AbdallahRdf/Main.java:1]: ok

File: submissions/aboullaite/Main.java [hunk 2]
```diff
@@ -1,65 +0,0 @@
-
-import java.io.IOException;
```
Comment (aboullaite) [inline submissions/aboullaite/Main.java:2]: Why you deleted this ? Please revert

**Review by aboullaite — CHANGES_REQUESTED** [review]

File: submissions/aboullaite/Main.java (hunk 2 above)
Comment (AbdallahRdf) [reply submissions/aboullaite/Main.java:2]: i think it's fixed now

Comment (aboullaite) [reply submissions/aboullaite/Main.java:2]: I still see deleted file in your PrR (2 files changed)

**Pull request closed without merging**